src/dotnet-compile-fsc/Program.cs

```diff
@@ -12,15 +12,16 @@
 using Microsoft.DotNet.Cli.Compiler.Common;
 using Microsoft.DotNet.Cli.Utils;
 using Microsoft.DotNet.ProjectModel;
+using Microsoft.DotNet.ProjectModel.Resolution;
 using NuGet.Frameworks;
+using System.Reflection;
 
 namespace Microsoft.DotNet.Tools.Compiler.Fsc
 {
     public class CompileFscCommand
```

Comment:
Existing code here by enrico, trying to have as little impact on dotnet-compile-fsc as possible because we will be taking this out of the repo.